projects/laji/src/app/shared-modules/pill-list/pill-list.component.scss

```diff
@@ -0,0 +1,26 @@
+@import 'vars';
+
+.label {
+  display: inline-block;
+  margin-top: $sp-1;
+  margin-right: $sp-1;
+
+  font-size: $fs-1;
+  color: $success-1;
+  background-color: $success-5;
+
+  box-shadow: 0 1px 2px 0 rgba(#000, .2);
+
+  .glyphicon-remove {
+    margin-left: $sp-2;
+    font-size: 10px;
+    padding: 3px;
+    color: $success-4;
+    background-color: $success-6;
+    border-radius: 1000px;
```

Comment:
`border-radius: 100%` would little nicer

Reply:
Hmm in this case either px or % units work just fine, because the height and width are equal. `border-radius` actually works quite differently with px and % units and `border-radius: 1000px;` is a standard instruction for styling rectangles with fully rounded corners (% units don't work for this). In this case maybe % units would be more intuitive 🤔